marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/models/updateModelInfoAndNamespace.api

```diff
@@ -0,0 +1,24 @@
+{
+  "functionName" : "updateModelInfoAndNamespace",
```

Comment:
updateModelInfo is fine and really preferable, since there technically is no such thing as a "model namespace", there's only a "definition namespace".